README.md

```diff
@@ -103,25 +104,25 @@ Bleachbit is a master at finding and removing temporary files, while Czkawka onl
 |         Language         |    Rust     |   Python   |   Python/Obj-C    |   Python    |
 |            OS            | Lin,Mac,Win |    Lin     |    Lin,Mac,Win    | Lin,Mac,Win |
 |        Framework         |    GTK 4    |   PyGTK2   | Qt 5 (PyQt)/Cocoa |   PyGTK3    |
-|     Duplicate finder     |      •      |     •      |         •         |             |
-|       Empty files        |      •      |     •      |                   |             |
-|      Empty folders       |      •      |     •      |                   |             |
-|     Temporary files      |      •      |     •      |                   |      •      |
-|        Big files         |      •      |            |                   |             |
-|      Similar images      |      •      |            |         •         |             |
-|      Similar videos      |      •      |            |                   |             |
-|  Music duplicates(tags)  |      •      |            |         •         |             |
-|     Invalid symlinks     |      •      |     •      |                   |             |
-|       Broken files       |      •      |            |                   |             |
-|      Names conflict      |      •      |     •      |                   |             |
-| Invalid names/extensions |      •      |     •      |                   |             |
-|    Installed packages    |             |     •      |                   |             |
-|          Bad ID          |             |     •      |                   |             |
-|  Non stripped binaries   |             |     •      |                   |             |
-|   Redundant whitespace   |             |     •      |                   |             |
-|    Overwriting files     |             |     •      |                   |      •      |
-|    Multiple languages    |      •      |     •      |         •         |      •      |
-|      Cache support       |      •      |            |         •         |             |
+|     Duplicate finder     |     ✔     |    ✔     |        ✔        |             |
+|       Empty files        |     ✔     |    ✔     |                   |             |
+|      Empty folders       |     ✔     |    ✔     |                   |             |
+|     Temporary files      |     ✔     |    ✔     |                   |     ✔     |
```

Comment:
Columns needs to have same width as before(constant)

Reply:
it has no influence or importance.
look at the final file 
https://github.com/qarmin/czkawka/blob/2a0be42eaef7928f2ee5d3bd3e0edb4b7b17d559/README.md

Reply:
@Fr-Dae It's not a matter of the final render, the file should have consistent style.

Reply:
github's display management does not take into account the number of spaces you leave before or after the |
It makes no difference whatsoever.

If you want to modify the readme, go ahead.
In any case when I edit it on my side all is correctly aligned.
Do you use a unicode font?